internal/flowcontrol/flow_control_manager.go

```diff
@@ -13,32 +13,31 @@ import (
 )
 
 type flowControlManager struct {
-	connParams             handshake.ParamsNegotiator
 	rttStats               *congestion.RTTStats
 	maxReceiveStreamWindow protocol.ByteCount
 
 	streamFlowController map[protocol.StreamID]*flowController
 	connFlowController   *flowController
 	mutex                sync.RWMutex
+
+	initialStreamSendWindow protocol.ByteCount
 }
 
 var _ FlowControlManager = &flowControlManager{}
 
-var errMapAccess = errors.New("Error accessing the flowController map.")
+var errMapAccess = errors.New("Error accessing the flowController map")
```

Comment:
Nit: (because you're touching it): error strings shouldn't be capitalized.

Reply:
You're right, but I won't fix this, since the `FlowControlManager` will be gone as soon as I merge #881 anyway.